tests/test_butler.py

```diff
@@ -698,8 +711,10 @@ def testConstructor(self) -> None:
                     uri = Butler.get_repo_uri("label")
                     butler = Butler.from_config(uri, writeable=False)
                     self.assertIsInstance(butler, Butler)
+                    butler.close()
```

Comment:
The logic here being that if it fails the assert above then the `close` here won't work anyhow.